pkg/operator/encryption/controllers/key_controller.go

```diff
@@ -252,7 +265,7 @@ func (c *keyController) generateKeySecret(keyID uint64, currentMode state.Mode,
 		InternalReason: internalReason,
 		ExternalReason: externalReason,
 	}
-	return secrets.FromKeyState(c.component, ks)
+	return secrets.FromKeyState(c.instanceName, ks)
```

Comment:
This change adjusts the name of secrets we create, which (I think) will impact upgrade and downgrade with this feature one.  Please keep this as `c.component`.

Reply:
> This change adjusts the name of secrets we create, which (I think) will impact upgrade and downgrade with this feature one. Please keep this as `c.component`.

Oh, I see. The name of the struct var changed, but the value did not.